functions.php

```diff
@@ -117,7 +117,7 @@ function send_email($email, $content, $subject, $append_unsubscribe=false, $psql
 
 	$content = wordwrap($content, 79, "\r\n");
 	$headers = "From: newsletter@nixers.net";
-	//print $email.":".$content;
+	print $email.":".$content;
```

Comment:
You may want to keep that commented.
I've only left it for debugging.
If you print that on the screen anyone will be able to see the content of the email sent and thus sign you up without replying/receiving the email.